docs: update layout of campaign creation section

vincentdelmundo opened the pull request:
# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [x] I wrote/updated tests for new/changed code
- [x] I ran `./gradlew check` without errors

corycaywood commented:
Ah, sorry I forgot about something. You will need to add this to the top of the `USERGUIDE.md` file to ensure that it uses the correct template when built by github pages:

```
---
layout: userguide
---
```

See this example: https://github.com/rakutentech/android-miniapp/blob/master/miniapp/USERGUIDE.md

vincentdelmundo commented:
> Ah, sorry I forgot about something. You will need to add this to the top of the `USERGUIDE.md` file to ensure that it uses the correct template when built by github pages:
> 
> ```
> ---
> layout: userguide
> ---
> ```
> 
> See this example: https://github.com/rakutentech/android-miniapp/blob/master/miniapp/USERGUIDE.md

Updated userguide.
And added code cov and ci result in readme

corycaywood commented:
Hmm, I tried running locally and something isn't working correctly: the home page is showing a directory listing instead of the userguide. Looks like the documentation script is missing this line: 
https://github.com/rakutentech/android-miniapp/blob/master/gradle/documentation.gradle#L29

Could you update this in the build-config script?
